src/scalarstats.jl

```diff
@@ -411,7 +411,7 @@ a precomputed mean `m`. The coefficient of variation is the ratio of the
 standard deviation to the mean.
 """
 variation(x, m) = stdm(x, m) / m
```

Comment:
This should be changed as well, I assume:
```suggestion
variation(x, m; corrected::Bool=true) = stdm(x, m; corrected=corrected) / m
```

Reply:
I found `stdm` to be in `deprecetes.jl` as of November 2021. So I was wondering if I need to change it or not...

Reply:
Only specific methods of `stdm` with weights are deprecated. The function that is called here is defined in the Statistics (https://docs.julialang.org/en/v1/stdlib/Statistics/#Statistics.stdm) and not (yet) deprecated there (https://github.com/JuliaStats/Statistics.jl/issues/5).

Reply:
Alright. I have made the required changes. Thanks.